smash-shells.lic

```diff
@@ -0,0 +1,67 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#smash-shells
+=end
+
+custom_require.call(%w[events common-items common-travel common])
+
+Flags.add('shell-fragments', 'shell explodes')
+
+junk_list = get_settings.pumpkin_junk.map { |x| /\b#{x}/i }
+
+DRCT.walk_to(16_236)
+loop do
+  break if checkbleeding
+  break if checkhealth < 50
+
+  until checkright || checkleft
+    DRC.fix_standing
+    case DRC.bput('grab box', 'roundtime', 'You are too injured', 'Perhaps that would work better', 'when you have enough coins', 'You should clean your hands')
+    when 'You are too injured', 'You should clean your hands'
+      DRCT.walk_to(16_150)
+      exit
+    when 'when you have enough coins'
+      DRCT.walk_to(16_315)
+      case DRC.bput('withdraw 3 platinum', 'we are not lending money at this time', 'The clerk counts out')
+      when 'we are not lending money at this time'
+        exit
```

Comment:
💡 Good place to tell the user they need to get more money

Reply:
Added message